### DISCUSSION TITLE
Blocking ad scripts in WebOne

### DISCUSSION THREAD

**Discussion by EgorKuzevanov** [atauenis/webone#69] [Ideas]
How to block ad scripts in WebOne that slow down the loading of some pages in older browsers, in some cases causing browsers crashes? The following domains (including subdomains) need to be blocked:

- googlesyndication.com
- an.yandex.ru
- mc.yandex.ru
- yastatic.net
- rareru.ru
- ad.mail.ru
- betweendigital.com
- marketgid.com
- utarget.ru
- doubleclick.net

**Comment by atauenis** [comment]
Read https://github.com/atauenis/webone/wiki/Sets-of-edits and create a rule which will redirect anything from these domains to a fake URL.

In v0.10.7 (or 0.11, or 0.10.8, currently don't know) i'm want to add some ad-blocking list support which will be easier than long Edit Sets which currently are the one way to implement the ad-banning.

**Comment by atauenis** [comment]
I have included such list in WebOne 0.10.7. It is disabled by default (as can break some sites), but it can be easily enabled by uncommenting its lines.

Seems that there is need to implement support for AdBlock Plus files (like EasyList, RuAdList, etc). But I currently have no time, so don't know when do this. If anyone will made a pull-request with the code, I'll be glad. :)
